third-party/gasnet/GASNet-1.26.0/other/amudp/amudp_spmd.cpp

```diff
@@ -565,18 +565,25 @@ extern int AMUDP_SPMDStartup(int *argc, char ***argv,
       //
       // Create a new argv with space for -E options for the env vars.
       //
-      int new_argc = 4 + *argc;
+      int new_argc = 4 + *argc + 1;
       char** new_argv = (char**) AMUDP_malloc(new_argc * sizeof((*new_argv)));
 
       //
       // Instead of running the program directly, run an xterm which
-      // runs gdb on the program and its args.
+      // runs lldb on the program and its args.
       //
+#if defined __APPLE__
+      new_argv[0] = (char *) "/opt/X11/bin/xterm";
+      new_argv[1] = (char *) "-e";
+      new_argv[2] = (char *) "lldb";
```

Comment:
I just did a `brew install gdb` and then maybe had to `brew link` it as well. 

I don't think you can rely on the path to gdb being the same across systems. But it should always be in the users path somewhere. 

I think I would just make users choose instead of trying to guess which one they want to use. i.e. have `CHPL_COMM_USE_GDB` and `CHPL_COMM_USE_LLDB` or have a `CHPL_COMM_USE_DB={GDB, LLDB}`